string ID
sensor_msgs/Image frame
